repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -3357,57 +3323,84 @@ public List<String> deleteClassificationPropagation(String entityGuid, String cl
         }
     }
 
-    public void deleteClassificationOnlyPropagation(Set<String> deletedEdgeIds) throws AtlasBaseException {
+    public void deleteClassificationOnlyPropagation(Set<String> deletedEdgeIds, String classificationVertexId) throws AtlasBaseException {
```

Comment:
implementation is different now in deleteClassificationOnlyPropagation function. Need to update this in updated code which is deployed in META-3141

Reply:
Please pull latest change and then modify